### PR TITLE
Make Footer more responsive and fix issues with Product Hero

### LINKED ISSUE #149: on firefox the upper part of the website is cut
Screenshot from 2022-07-20 13-16-41

### LINKED ISSUE #181: Increase spacing between logos in the footer

### LINKED ISSUE #244: Get rid of unnecessary console logs
There are a bunch of console errors/warnings when we load different pages of the app. Listing them below so we can fix them if we're working on any of these pages or if we have time:

1. ProductHeroLayout. Example:
`mergeClasses.js?5cd6:25 Material-UI: The key `h2` provided to the classes prop is not implemented in ProductHeroLayout.
You can only override one of the following: root,container,backdrop,background,arrowDown,imageLicenseContainer,imageLicense.`
2.

### LINKED ISSUE #269: Make Footer more responsive and fix ProductHero Layout
The app footer is not responsive (breaks the UI when the browser's size is changed).

There are also a lot of issues with the product hero Layout and it also breaks the UI in different modes.

Screen Shot 2023-04-03 at 9 03 57 PM

Screen Shot 2023-04-03 at 9 03 08 PM

### PR DESCRIPTION
**Pull request by sami1riaz**
Adding screenshots of what the UI looks like now but I'd suggest cloning the branch and checking it out yourself as well to see if there are any issues.

Screen Shot 2023-04-03 at 9 07 17 PM

Screen Shot 2023-04-03 at 9 07 34 PM
(I know that the text section here looks a bit bland but I plan on improving that. Was hoping to have a design discussion with the team before changing it. We could also probably look into changing our homepage picture)

Note: Ignore the name of the branch, I was initially working on issue 250 and got distracted which led to this PR

closes #269, #244 & #181

### CODE REVIEW

File: frontend/src/views/App/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    width: 80,
+    height: 50,
+  },
+  supporterLogo: {
+    width: 48,
+    height: 48,
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    margin: "10px",
```
Comment (veitu) [inline frontend/src/views/App/Footer.tsx]: Looks very nice, thank you! If possible maybe you could increase the spacing between the supporter logos? (At least between Goethe Institute and GIScience logos) As I'm not familiar with this code, I will leave further review up to the people wo are familiar with it :)

Comment (sami1riaz) [reply frontend/src/views/App/Footer.tsx]: yeah i'll increase the spacing. good catch

File: frontend/pages/index.tsx [hunk 2]
```diff
@@ -1,101 +1,73 @@
...
-import { Typography, makeStyles, Theme, Button } from "@material-ui/core";
-
-
-const backgroundImage = '../static/images/background_pic.jpg'
+import Hero from "../src/components/Hero/Hero";
+import Footer from "../src/views/App/Footer";
 
+const backgroundImage = "../static/images/background_pic.jpg";
+const subscribeForm =
+  "https://docs.google.com/forms/d/e/1FAIpQLSc5Lj8i0pWIlLU58jycqAlaPSGYYrPJgPUqm1l3Hw_60yOSYA/viewform?usp=sf_link";
```
Comment (codingfabi) [inline frontend/pages/index.tsx:13]: Is this some leftover code or do we actually need this?

**Review by codingfabi — APPROVED** [review]
Looks good, the new footer component is really nice ;)

**Pull request merged**